live-examples/js-examples/number-epsilon.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">x = 0.2;
+y = 0.3;
+z = 0.1;
```

Comment:
Missing `var` for the above three lines.

Reply:
OK